### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Aplicticus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# ShipServiceManagementService
```
Comment (markieo1) [inline README.md:1]: No need for change

File: ShipServiceManagement.Logic/Extensions/DIHelper.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+
+namespace ShipServiceManagement.Logic.Extensions
+{
+	public static class DIHelper
+    {
+		public static IServiceCollection AddLogic(this IServiceCollection services)
```
Comment (markieo1) [inline ShipServiceManagement.Logic/Extensions/DIHelper.cs:7]: Why do we have this file?

File: ShipServiceManagement.App/Controllers/ShipServiceController.cs [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+		{
+			IActionResult response = null;
+
+			if (string.IsNullOrEmpty(name))
+			{
+				response = NotFound();
+			}
+			else
+			{
+				_shipServiceManager.DeleteShipService(name);
```
Comment (markieo1) [inline ShipServiceManagement.App/Controllers/ShipServiceController.cs:83]: await this call

File: ShipServiceManagement.App/Controllers/ShipServiceController.cs [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+		{
+			IActionResult response = null;
+
+			if (shipService == null)
+			{
+				response = NotFound();
+			}
+			else
+			{
+				_shipServiceManager.UpdateShipService(shipService);
```
Comment (markieo1) [inline ShipServiceManagement.App/Controllers/ShipServiceController.cs:60]: await this call

File: ShipServiceManagement.App/Controllers/ShipServiceController.cs [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+		{
+			IActionResult response = null;
+
+			if (shipService == null)
+			{
+				response = NotFound();
+			}
+			else
+			{
+				_shipServiceManager.CreateShipService(shipService);
```
Comment (markieo1) [inline ShipServiceManagement.App/Controllers/ShipServiceController.cs:37]: await this call

File: ShipServiceManagement.Logic/ShipServiceManagement.Logic.csproj [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+  <ItemGroup>
+    <ProjectReference Include="..\ShipServiceManagement.Messaging\ShipServiceManagement.Messaging.csproj" />
+    <ProjectReference Include="..\ShipServiceManagement.Models\ShipServiceManagement.Models.csproj" />
+    <ProjectReference Include="..\ShipServiceManagement.Persistence\ShipServiceManagement.Persistence.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.Extensions.DependencyInjection.Abstractions">
+      <HintPath>C:\Program Files\dotnet\sdk\NuGetFallbackFolder\microsoft.extensions.dependencyinjection.abstractions\2.0.0\lib\netstandard2.0\Microsoft.Extensions.DependencyInjection.Abstractions.dll</HintPath>
```
Comment (markieo1) [inline ShipServiceManagement.Logic/ShipServiceManagement.Logic.csproj]: Make use of PackageReference / Nuget

File: ShipServiceManagement.Messaging/ShipServiceManagement.Messaging.csproj [hunk 7]
```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.Extensions.DependencyInjection.Abstractions">
+      <HintPath>C:\Program Files\dotnet\sdk\NuGetFallbackFolder\microsoft.extensions.dependencyinjection.abstractions\2.0.0\lib\netstandard2.0\Microsoft.Extensions.DependencyInjection.Abstractions.dll</HintPath>
```
Comment (markieo1) [inline ShipServiceManagement.Messaging/ShipServiceManagement.Messaging.csproj]: Make use of PackageReference / Nuget

**Review by markieo1 — CHANGES_REQUESTED** [review]

File: ShipServiceManagement.App/Controllers/ShipServiceController.cs (hunk 5 above)
Comment (Aplicticus) [reply ShipServiceManagement.App/Controllers/ShipServiceController.cs:37]: fixed by adding Task and await

File: ShipServiceManagement.App/Controllers/ShipServiceController.cs (hunk 4 above)
Comment (Aplicticus) [reply ShipServiceManagement.App/Controllers/ShipServiceController.cs:60]: fixed by adding Task and await

File: ShipServiceManagement.App/Controllers/ShipServiceController.cs (hunk 3 above)
Comment (Aplicticus) [reply ShipServiceManagement.App/Controllers/ShipServiceController.cs:83]: fixed by adding Task and await

**Pull request closed without merging**